JapaneseFilter/sections/css_extended.txt

```diff
@@ -101,6 +101,7 @@ kijonotakuhaibin.com#?#.plugin-memo:has(> div.side > div[id^="side-koukoku"])
 domani.shogakukan.co.jp#?#.textwidget:has(> div[id^="div-gpt-ad"])
 kijomatomelog.com#?#.sub_right > .sidewrapper:has(> .sidetitlebody > .sidetitle:contains(スポンサードリンク))
 asahi.com#?#.p-kurashi__list > div.p-kurashi__listItem:has(> div[id^="div-gpt-ad-"])
+hinode.pics#?##right_wrap > div.right_plugin_wrap:contains(広告)
```

Comment:
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_title:contains(広告))
```
You should specify the element which contains it.
However, I'd prefer to use the rule without `:contains()`.
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_body > div[style] > ins.adsbygoogle)
```

Reply:
Thank you